### PR TITLE
Initial file for kathie bio

### PR DESCRIPTION
**Pull request by kathiefoley**
Adding a file to address #53

### CODE REVIEW

File: bios/kathie-bio.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## A little about me
+- I live in San Diego
```
Comment (kathiefoley) [inline bios/kathie-bio.md]: Where in San Diego do you live?

File: bios/kathie-bio.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## A little about me
+- I live in Scripps Ranch, San Diego
+- I am married, two kids, one dog
```
Comment (c5chen) [inline bios/kathie-bio.md:3]: Can you tell us about your hobbies?

Comment (kathiefoley) [inline bios/kathie-bio.md:3]: Good idea!

**Comment by kathiefoley** [comment]
:+1: This is ready to go

**Pull request merged**